link error master

Hi, I wonder how this slipped through, but seems like the last PR did not compile?

```
/elektra-clang/build/src/plugins/fcrypt    -fPIC -o CMakeFiles/elektra-fcrypt-objects.dir/__/crypto/helper.c.o   -c /home/jenkins/workspace/workspace/elektra-clang/src/plugins/crypto/helper.c
/home/jenkins/workspace/workspace/elektra-clang/src/plugins/crypto/helper.c:13:10: fatal error: 'base64_functions.h' file not found
#include <base64_functions.h>
         ^
```

--- a/src/plugins/fcrypt/CMakeLists.txt
+++ b/src/plugins/fcrypt/CMakeLists.txt
@@ -2,6 +2,8 @@ include (LibAddPlugin)
 
 add_plugin (fcrypt
 	SOURCES
+		../base64/base64_functions.h
+		../base64/base64_functions.c
 		../crypto/helper.h
 		../crypto/helper.c
 		../crypto/gpg.h
@@ -10,8 +12,10 @@ add_plugin (fcrypt
 		fcrypt.c
 	INCLUDE_DIRECTORIES
 		${CMAKE_SOURCE_DIR}/src/plugins/crypto/
+		${CMAKE_SOURCE_DIR}/src/plugins/base64/
 	COMPILE_DEFINITIONS
 		ELEKTRA_PLUGIN_NAME=\"fcrypt\"
+		ELEKTRA_PLUGIN_NAME_C=fcrypt
 )
 
 if (ADDTESTING_PHASE)

